### ISSUE TITLE
[Question] Is it possible to change the mesh of the object after every environment step?

### ISSUE THREAD

**Issue by chandramohanjagadeesan** [isaac-sim/IsaacLab#1063]
I am working on a continual Reinforcement learning project where I want more objects appearing or disappearing in the environment. Is there a way to make it possible in Isaaclab? Ideally I would like to change the mesh of the object from one time step to another. Thanks in advance for your reply

**Comment by garylvov** [comment]
I don't believe this is directly supported, but you can certainly emulate this behavior.

I'd say that the closest thing to what you want is: https://github.com/isaac-sim/IsaacLab/issues/186

Some hacky workarounds you could try:
Spawn all of the objects of interest for each environment, and then "teleport" the objects you don't care about as far from the robot (say, 10m), and the objects you care about close to the robot.
